Specification/4._CAN_bus_transport_layer.md

```diff
@@ -315,64 +311,65 @@ Source node ID                  | 7    | 1...127        | See below
 
 #### Priority
 
-In multi-frame transfers, the value of the priority field must be identical for all frames of the transfer.
-As can be seen from the description, valid values for priority range from 0 to 31, inclusively,
-where 0 corresponds to highest priority, and 31 corresponds to lowest priority.
+Valid values for priority range from 0 to 31, inclusively, where 0 corresponds to highest priority (and 31 corresponds to lowest priority).
+
+In multi-frame transfers, the value of the *priority* field must be identical for all frames of the transfer.
 
 #### Message type ID
 
-As can be seen from the description, valid values of message type ID range from 0 to 65535, inclusively.
+Valid values of message type ID range from 0 to 65535, inclusively.
```

Comment:
@pavel-kirienko I globally removed "as can be seen from the description" because IMO there are two steps to getting the information from the description - counting the bits available, then working out the size of data available (ie not obvious on very first glance). I think the way it is now is better - they see the valid values, and working out why they are so is then easy.

Reply:
Okay.